src/neo/SmartContract/Helper.cs

```diff
@@ -30,87 +30,17 @@ public static class Helper
-                {
-                    case OpCode.JMP:
-                    case OpCode.JMPIF:
-                    case OpCode.JMPIFNOT:
-                    case OpCode.JMPEQ:
-                    case OpCode.JMPNE:
-                    case OpCode.JMPGT:
-                    case OpCode.JMPGE:
-                    case OpCode.JMPLT:
-                    case OpCode.JMPLE:
-                    case OpCode.CALL:
-                    case OpCode.ENDTRY:
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI8)))
-                            return false;
-                        break;
-                    case OpCode.PUSHA:
-                    case OpCode.JMP_L:
-                    case OpCode.JMPIF_L:
-                    case OpCode.JMPIFNOT_L:
-                    case OpCode.JMPEQ_L:
-                    case OpCode.JMPNE_L:
-                    case OpCode.JMPGT_L:
-                    case OpCode.JMPGE_L:
-                    case OpCode.JMPLT_L:
-                    case OpCode.JMPLE_L:
-                    case OpCode.CALL_L:
-                    case OpCode.ENDTRY_L:
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI32)))
-                            return false;
-                        break;
-                    case OpCode.TRY:
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI8)))
-                            return false;
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI8_1)))
-                            return false;
-                        break;
-                    case OpCode.TRY_L:
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI32)))
-                            return false;
-                        if (!instructions.ContainsKey(checked(ip + instruction.TokenI32_1)))
-                            return false;
-                        break;
-                    case OpCode.NEWARRAY_T:
-                    case OpCode.ISTYPE:
-                    case OpCode.CONVERT:
-                        StackItemType type = (StackItemType)instruction.TokenU8;
-                        if (!Enum.IsDefined(typeof(StackItemType), type))
-                            return false;
-                        if (instruction.OpCode != OpCode.NEWARRAY_T && type == StackItemType.Any)
-                            return false;
-                        break;
-                }
-            }
-            if (abi is null) return true;
-            foreach (ContractMethodDescriptor method in abi.Methods)
-            {
-                if (!instructions.ContainsKey(method.Offset))
-                    return false;
-            }
-            try
+            Script s = new Script(script, true);
+            if (abi is not null)
             {
+                foreach (ContractMethodDescriptor method in abi.Methods)
+                    s.GetInstruction(method.Offset);
```

Comment:
It can throw `ArgumentException` so we should add this type to `VerifyWitness`
https://github.com/neo-project/neo-vm/blob/df2cc9746e684ae877dd7b1a83d10a652d4c852c/src/neo-vm/Script.cs#L121

Reply:
Modified.